labapi/src/main/java/com/microsoft/identity/internal/test/labapi/api/LabSecretApi.java

```diff
@@ -138,7 +138,7 @@ public SecretResponse apiLabSecretGet(String secret) throws ApiException {
      */
     public ApiResponse<SecretResponse> apiLabSecretGetWithHttpInfo(String secret) throws ApiException {
         com.squareup.okhttp.Call call = apiLabSecretGetValidateBeforeCall(secret, null, null);
-        Type localVarReturnType = new TypeToken<SecretResponse>(){}.getType();
+        Type localVarReturnType = TypeToken.get(SecretResponse.class).getType();
```

Comment:
Ok, so most of the changes you are making here are actually inside LabApi code, which was auto generated via Swagger. So we're going to get whatever swagger generates. The next time we regenerate the client, most likely all changes you've made here will be lost and overwritten again with the unsafe TypeToken instances (given that it seems that's what Swagger does right now).